### ISSUE TITLE
ua start_register error

### ISSUE THREAD

**Issue by nanguantong** [baresip/baresip#1116]
ua.c

```
ua_event(ua, fallback ?
		UA_EVENT_REGISTER_FAIL :
		UA_EVENT_FALLBACK_FAIL,
		NULL, "%m", err);
====>
ua_event(ua, fallback ?
		UA_EVENT_FALLBACK_FAIL :
		UA_EVENT_REGISTER_FAIL,
		NULL, "%m", err);
```

**Comment by sreimers** [comment]
Good catch, would you like to prepare a Pull request?

**Comment by nanguantong** [comment]
Yeh, I commited pr, please review.

**Comment by alfredh** [comment]
fixed in 0b5f64d315cab5fbc7b5c1d133e905b361b8c89b

Thanks!
